docs/testing/applitools_automated_ui_visual_testing.md

```diff
@@ -28,9 +35,9 @@ The configuration file has been added to the `[tsconfig.cypress.json](./tsconfig
 
 In order run Cypress with Applitools, you will need an [Applitools](https://applitools.com/) account. Once signed up (there is a free tier), the API key can be used to create a connection with Applitools.
 
-Please follow the nvm[Cypress Tutorial](https://applitools.com/tutorials/cypress.html)
+Please follow the nvm [Cypress Tutorial](https://applitools.com/tutorials/cypress.html)
 
-`export APPLITOOLS_API_KEY=`
+`export APPLITOOLS_API_KEY=KEY_VALUE`
```

Comment:
Would be  easier for the user to copy it if this is a \<code\> (wrapper in ```).

Reply:
fixed